### ISSUE TITLE
Ground Zero game incomplete 

### ISSUE THREAD

**Issue by HellsBells24** [yquake2/rogue#76]
I'm using the flatpak of Yamagi Quake II to play the Ground Zero expansion pack. The pak0.pak and video files were taken from my Steam files and are in the rogue folder inside the folder for the flatpak (they've also previously been used to install the game on another computer). Despite this, I've noticed some issues that prevent the game from being played. The first secret is accessible but there is no power up present once you get there. There are no Tesla packs present, nor does the elevator in the first level work, making you unable to progress in the level. I'm not sure what other info is needed to help solve the problem, but please let me know!
Thank you!

**Comment by DanielGibson** [comment]
sounds like rogue/game.so is missing (and the game falls back to the default game.so of the main game which of course doesn't support the ground zero-specific entities)

**Comment by HellsBells24** [comment]
You are correct, I do not see a game.so file in the rogue directory, so I would assume that is the problem. Of course, it brings up more questions. I own both the Steam version and GoG version of Quake II, but do not see that file in either of those game files. I also played through The Reckoning with no issue, but see no such file there either. It seems odd to me. 
Anyway, any idea how to obtain this file?

**Comment by DanielGibson** [comment]
That file is compiled from this git repository.
Theoretically that flatpak could (should?) contain it but apparently it doesn't (note that we don't maintain that flatpak and this is the first time I even heard about a YQ2 flatpak).

You could try building it yourself, I think you'll only need make and gcc installed.
Download the source of this repository and run `make -j4` in its root directory. When that's done, game.so should be in the release/ subdirectory and you can copy it to your installation

**Comment by HellsBells24** [comment]
Ah, I'm very sorry, the Flathub "Help" entry leads back here and it was my assumption that you did maintain it. 
I came up with a creative solution though. I'm not able to run YQ2 in the traditional manner on my PopOS desktop (hence the flatpak), but Kubuntu 20.04 runs it just fine. And as a result, the game.so file was located!
I transferred the file, ran it via Flatpak, and picked up the Double Damage power up.
Thank you and once again, sorry to bother you about the flatpak
